Gopkg.lock

```diff
@@ -285,6 +279,14 @@
   revision = "1624edc4454b8682399def8740d46db5e4362ba4"
   version = "v1.1.5"
 
+[[projects]]
```

Comment:
why do we need this? How did it end up in the lock file?

Reply:
Hm, I just ran `dep ensure -v`, I will check the dependency tree to see from where it is coming.

Reply:
Ok, `logrus` v1.3.0 has this dependency

Reply:
definitely sg